src/test/resources/cucumber/features/Modernisation.jurorResponses/JM-6060.feature

```diff
@@ -0,0 +1,155 @@
+    And I press the "Save" button
+    Then I see error "The notes provided are too long"
+    And I see "You have 1 character too many" on the page
+    When I set "Add a note" to "This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 characters long. This note is 2000 characters long This note is 2000 characters long. This note is 2000 charac"
```

Comment:
just a suggestion-Instead of adding big  note like this, create a java method to add string of characters" and pass to field